advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py

```diff
@@ -303,13 +304,15 @@ def __init__(self, context):
 
         # setting up the hand selection
         hand_finder = HandFinder()
-        self.hand_parameters = hand_finder.get_hand_parameters()
+        if hand_finder.hand_e_available():
```

Comment:
Maybe put everything down to line 317 inside the  same ```if hand_finder.hand_e_available()``` and add a todo to extend to choosing work with Hand H too.